package.json

```diff
@@ -679,14 +674,24 @@
           "group": "navigation@2"
         },
         {
-          "command": "confluent.resources.schema-registry.select",
+          "command": "confluent.schemas.create",
           "when": "view == confluent-schemas",
           "group": "navigation@1"
         },
+        {
+          "command": "confluent.schemas.upload",
+          "when": "view == confluent-schemas && confluent.schemaRegistrySelected",
+          "group": "navigation@2"
+        },
```

Comment:
We could also extend this to `"when": "view == confluent-resources && viewItem =~ /.*schema-registry.*/"` so it's available as an action in the Resources view on a Schema Registry item

Reply:
Done in https://github.com/confluentinc/vscode/pull/741/commits/21eb5ba3552ca27fb5528f7040060f8f071bcb5e